.github/workflows/deploy-dev.yaml

```
@@ -0,0 +1,19 @@
+name: Deploy to Dev Environment
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mbta/actions/npm-install
+        with:
+          node-version: 14
+      - uses: mbta/actions/deploy-lambda
```

Oh! One thing I forgot – can you create a new [release in the actions repo](https://github.com/mbta/actions/releases) and re-tag the `v1` tag, and then reference the `v1` version here and in the prod workflow below? I can talk you through the process for cutting a new release and updating the tag.

```suggestion
      - uses: mbta/actions/npm-install@v1
        with:
          node-version: 14
      - uses: mbta/actions/deploy-lambda@v1
```

---

@ianwestcott Sure can we get on call when you are free to show me or make sure I'm doing it right as it involves retagging rather than assigning a new tag.

---

done